src/models/MyAccount.ts

```diff
@@ -113,11 +113,16 @@ export default class MyAccount {
         canFreeze: hold.canFreeze,
         frozen: hold.frozen,
         status: MyAccount.getHoldStatus(hold.status),
-        pickupLocation: hold.pickupLocation.name,
+        pickupLocation: hold.pickupLocation,
         title: bibDataMap[hold.record.bibIds[0]].title,
         isResearch: bibDataMap[hold.record.bibIds[0]].isResearch,
         bibId: hold.record.bibIds[0],
-        isNyplOwned: bibDataMap[hold.record.bibIds[0]].isResearch,
+        isNyplOwned: bibDataMap[hold.record.bibIds[0]].isNyplOwned,
```

Comment:
Yes, I think, as a convenience we only render the title of the first bib when an item is a bound-with. Ideally we would display all titles because the patron may have requested an item on one bib and be surprised to see the title of a different bib in their holds/checkouts. Would be better to show all titles even crudely (e.g. `hold.record.bibIds.map((id) => bibDataMap[id].title).join('; ')`)

For the purpose of determining "is-nypl-owned", just checking the first bib is enough because an item will never contain a mix of institution bibs.